config/src/config/quorum_store_config.rs

```diff
@@ -39,23 +63,21 @@ impl Default for QuorumStoreConfig {
             channel_size: 1000,
             proof_timeout_ms: 10000,
             batch_request_num_peers: 2,
-            mempool_pulling_interval: 100,
-            end_batch_ms: 500,
-            max_batch_counts: 300,
-            max_batch_bytes: 1000000,
+            mempool_pulling_interval: 250,
+            // TODO: This essentially turns fragments off, because there was performance degradation. Needs more investigation.
+            end_batch_ms: 10,
```

Comment:
because back pressure slows down the batch creation?

Reply:
It shouldn't necessarily, if we're asking for the same number of transactions on the pull. But I was seeing more than one fragment being created. I'll try to revive fragments in the next PR.